db/silkworm/stagedsync/stage_log_index.cpp

```diff
@@ -153,8 +153,70 @@ StageResult stage_log_index(TransactionManager &txn, const std::filesystem::path
     return StageResult::kSuccess;
 }
 
-StageResult unwind_log_index(TransactionManager &, const std::filesystem::path &, uint64_t) {
-    throw std::runtime_error("Not Implemented.");
-}
+StageResult unwind_log_index(TransactionManager& txn, etl::Collector &collector, uint64_t unwind_to, bool topics) {
```

Comment:
Better mark it `static` to enable [Internal Linkage](https://google.github.io/styleguide/cppguide.html#Internal_Linkage) since this function is not declared in the header and is not used outside `stage_log_index.cpp`.